### PR TITLE
More assorted ROMs.

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

File: entries/rgb-grafx-demo/game.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "files": [
+        {
+            "default": true,
+            "filename": "RGB Grafx Demo (PD) [C].gbc",
+            "playable": true
+        }
+    ],
+    "platform": "GBC",
+    "screenshots": [
+        "RGB Grafx Demo (PD) [C].png"
```
Comment (avivace) [inline entries/rgb-grafx-demo/game.json:12]: I think this one should be retaken manually

File: entries/segmond-first-demo/game.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "files": [
+        {
+            "default": true,
+            "filename": "Segmond First Demo (PD).gb",
+            "playable": true
+        }
+    ],
+    "platform": "GB",
+    "screenshots": [
+        "Segmond First Demo (PD).png"
```
Comment (avivace) [inline entries/segmond-first-demo/game.json:12]: To be retaken manually (the screenshot)

File: entries/xevious-millenium/Xevious Millennium (PD) [C].sav [hunk 3]
```diff
@@ -0,0 +1 @@
...������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������
```
Comment (avivace) [inline entries/xevious-millenium/Xevious Millennium (PD) [C].sav]: Should the sav be here?

**Review by avivace — CHANGES_REQUESTED** [review]

File: entries/rgb-grafx-demo/game.json (hunk 1 above)
Comment (asiekierka) [reply entries/rgb-grafx-demo/game.json:12]: Last I checked, that was literally all the demo was

File: entries/segmond-first-demo/game.json (hunk 2 above)
Comment (asiekierka) [reply entries/segmond-first-demo/game.json:12]: Likewise, this is all the demo is AFAIK

File: entries/xevious-millenium/Xevious Millennium (PD) [C].sav (hunk 3 above)
Comment (asiekierka) [reply entries/xevious-millenium/Xevious Millennium (PD) [C].sav]: Already removed

**Review by avivace — APPROVED** [review]

**Pull request merged**